### ISSUE TITLE
After: Relaunch Accessibility

### ISSUE THREAD

**Issue by leonard-haas** [aula-app/aula-frontend#240]
[Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) 2.1

**Comment by eduardoliron** [comment]
@leonard-haas, I tried to keep the work explicit on the commits: https://github.com/aula-app/aula-frontend/pull/646

In short:
Added Ids, ARIA and role definitions to components
Added translation to ARIA descriptions
Improved overall keyboard navigation
Added focus indicators for keyboard navigation
Added meaningful alt text to images

**Comment by eduardoliron** [comment]
Merged into DEV

**Comment by nikola-maric-aula** [comment]
i confirmed the changes on devel
